### ISSUE TITLE
[FormBody] System.IO.Stream stream is producing "format": "byte" rather than "format": "binary"

### ISSUE THREAD

**Issue by rebeccapowell** [RSuter/NJsonSchema#822]
DELETED

**Comment by rebeccapowell** [comment]
I have now switched to the new interface, but the problem is the same. I'm starting to think I read https://github.com/RSuter/NJsonSchema/issues/445#issuecomment-321240548 incorrectly?

I also notice that AddMissingPathParameters isn't required i the new generation format and it works out the basecontroller route anyway. One thing that is different and I'm not sure if it an issue, is that the query parameters I've wrapped in an object, so that I can add a required attribute to certain parameters:

```
public class FileQueryParameters
{
	[BindRequired]
	public string Filename { get; set; }

	[BindRequired]
	public DocumentType DocumentType { get; set; }
}
```

In the UseSwaggerUi3 generation these parameter names are now upper case, rather than lower case as they were before:

```
{
	"type": "string",
	"name": "Filename",
	"in": "query",
	"required": true,
	"x-nullable": true
}, {
	"type": "integer",
	"name": "DocumentType",
	"in": "query",
	"required": true,
	"x-schema": {
		"$ref": "#/definitions/DocumentType"
	},
	"x-nullable": false,
	"enum": [
		0,
		1,
		2,
		3
	]
},
```
I've also tried both:

`[Consumes("application/octet-stream", "application/pdf", "image/jpg", "image/jpeg", "image/png", "image/tiff", "image/tif")]`
	
and

`[Consumes("application/octet-stream", new string[] { "application/pdf", "image/jpg", "image/jpeg", "image/png", "image/tiff", "image/tif"})]`

Neither seems to make a difference. Notably, posting a file also rejects any Content-Type that isn't "application/octet-stream", so perhaps the `ConsumesAttribute` doesn't support multiple MIME types.
